src/part08.rs

```diff
@@ -48,21 +55,28 @@ fn test_overflowing_add() {
 }
 
 // ## Associated Types
-//@ Now we are equipped to write the addition function for `BigInt`. As you may have guessed, the `+` operator
-//@ is tied to a trait (`std::ops::Add`), which we are going to implement for `BigInt`.
+//@ Now we are equipped to write the addition function for `BigInt`. As you may have guessed, the
+//@ `+` operator is tied to a trait (`std::ops::Add`), which we are going to implement for
+//@ `BigInt`.
 //@ 
-//@ In general, addition need not be homogeneous: You could add things of different types, like vectors and points. So when implementing
-//@ `Add` for a type, one has to specify the type of the other operand. In this case, it will also be `BigInt` (and we could have left it
-//@ away, since that's the default).
+//@ In general, addition need not be homogeneous: You could add things of different types, like
+//@ vectors and points. So when implementing `Add` for a type, one has to specify the type of the
+//@ other operand. In this case, it will also be `BigInt` (and we could have left it away, since
+//@ that's the default).
 impl ops::Add<BigInt> for BigInt {
-    //@ Besides static functions and methods, traits can contain *associated types*: This is a type chosen by every particular implementation
-    //@ of the trait. The methods of the trait can then refer to that type. In the case of addition, it is used to give the type of the result.
-    //@ (Also see the [documentation of `Add`](https://doc.rust-lang.org/stable/std/ops/trait.Add.html).)
+
```

Comment:
Another one. I guess I will stop marking them here :)

Reply:
Yeah, sorry again. Feel free to change my patch however you see fit.

Reply:
Any chance you could try and fix the empty-line-changes? There's no time pressure anyway.^^

Reply:
Sure, it'd help if you would indeed mark them when you can.